documentation/server_options.md

```diff
@@ -20,8 +20,8 @@ Being mindful of usage helps ensure that the platform remains sustainable and ac
 Please follow these guidelines:
 
 **Start small:** Begin by testing your workflow on the smallest server that meets your needs.
-Limit initial runs in scope (e.g., fewer sources, shorter iterations, smaller datasets).
```

Comment:
I'm pretty sure that the comma after "e.g." is grammatically correct. For example, https://www.grammarly.com/blog/acronyms-abbreviations/i-e-vs-e-g/#4.

Reply:
FWIW, I would love to enforce of not having bad latin, and have a script and GHA to do so, you all just say the words :) 

https://www.gov.uk/guidance/style-guide/a-to-z#eg-etc-and-ie

or see for reasons: https://book.the-turing-way.org/community-handbook/style/consistency/consistency-language#ch-consistency-language-hr-abbreviations and https://book.the-turing-way.org/community-handbook/style/style-writing-markdown#avoid-latin-abbreviation